gp-includes/things/translation-set.php

```diff
@@ -267,9 +267,15 @@ public function update_status_breakdown() {
 			 */
 			$t = GP::$translation->table;
 			$o = GP::$original->table;
+
+			$maybe_exclude_hidden = '';
+			if ( apply_filters( 'gp_exclude_hidden_originals_from_counts', false ) ) {
+				$maybe_exclude_hidden = "AND o.priority != '-2'";
+			};
+
 			$counts = GP::$translation->many_no_map("
 				SELECT t.status as translation_status, COUNT(*) as n
-				FROM $t AS t INNER JOIN $o AS o ON t.original_id = o.id WHERE t.translation_set_id = %d AND o.status LIKE '+%%' GROUP BY t.status", $this->id);
+				FROM $t AS t INNER JOIN $o AS o ON t.original_id = o.id WHERE t.translation_set_id = %d AND o.status LIKE '+%%' $maybe_exclude_hidden GROUP BY t.status", $this->id);
```

Comment:
@toolstack I don't think that matters. The join is done on specific original by id, and I'm pretty sure mysql is optimizing wether to check for the status or the priority first. I've run some `EXPLAIN` on this and didn't see any difference.